src/backend/api/Fusion.Resources.Logic/Requests/Commands/ResourceAllocationRequest/Allocation/ProvisionAllocationRequest.cs

```diff
@@ -105,40 +93,6 @@ private async Task AllocateRequestInstanceAsync(DbResourceAllocationRequest dbRe
                         if (!updateResp.IsSuccessStatusCode)
                             throw new OrgApiError(updateResp.Response, updateResp.Content);
                     }
-
-                    private async Task ProcessFutureInstancesAsync(DbResourceAllocationRequest dbRequest, ApiDraftV2 draft, ApiPositionV2 position)
```

Comment:
Just checking. Was removal of this intentional in this pull request?

Reply:
Yes, i belive it is. I think it was unclear what was going on, so to change this they wanted to create 1 request pr instance instead, hence the batch endpoint.